deploy/helm/kubernetes/templates/etcd-statefulset.yaml

```diff
@@ -103,6 +102,35 @@ spec:
       - secret:
           secretName: {{ $fullName }}-pki-etcd-server
         name: pki-etcd-certs-server
+        {{- if not .Values.persistence.enabled }}
```

Comment:
please move it on same level with item, eg:

```yaml
list:
{{- if condition }}
- some: key
{{- end }} 
```

not 

```yaml
list:
  {{- if condition }}
- some: key
  {{- end }} 
```

Reply:
see [35f002f](https://github.com/kvaps/kubernetes-in-kubernetes/pull/1/commits/35f002ff8255c6d56dca852fadaa9fbae9ea06de)